build.gradle.kts

```diff
@@ -90,6 +90,7 @@ dependencies {
       "org.jetbrains.kotlin",
       "org.jetbrains.plugins.gradle",
       "org.intellij.intelliLang",
+      "com.google.tools.ij.aiplugin",
```

Comment:
should this be in the next if statement, only if the product is android studio?

Reply:
If we can provide the support to all users without additional work we should.  It may be the case that some support is added through here, and other support will be AS specific.